rocketpy/plots/aero_surface_plots.py

```diff
@@ -448,3 +448,84 @@ def __init__(self, tail):
     def draw(self):
         # This will de done in the future
         return None
+
+
+class _AirbrakesPlots(_AeroSurfacePlots):
+    """Class that contains all airbrakes plots."""
+
+    def __init__(self, airbrakes):
+        """Initialize the class
+
+        Parameters
+        ----------
+        airbrakes : rocketpy.AeroSurface.airbrakes
+            Airbrakes object to be plotted
+
+        Returns
+        -------
+        None
+        """
+        super().__init__(airbrakes)
+        return None
+
+    def cd_curve(self):
+        return self.aero_surface.cd.plot(0, 1)
+
+    def deployed_level(self):
+        """Plots the deployed level of the airbrakes as a function of time.
+
+        Returns
+        -------
+        None
+        """
+        # Extract the first and second columns as x and y
+        x = [row[0] for row in self.aero_surface.state_history]
+        y = [row[1] for row in self.aero_surface.state_history]
+
+        # Create the plot
+        plt.figure(figsize=(10, 6))
+        plt.scatter(x, y, marker="o", linestyle="-", color="b", s=0.5)
+        plt.xlabel("Time (s)")
+        plt.ylabel("Deployed Level")
+        plt.title("Deployed Level X Time (s)")
+        plt.grid(True)
+        plt.show()
+
+        return None
+
+    def cd(self):
```

Comment:
isn't `cd` a too short name for the method? 
Did you consider using `drag_coefficient` instead?